deploy/terraform-gcp/README.md

```diff
@@ -27,17 +27,17 @@ This will create a new GKE cluster and deploy Kip:
     $ kubectl get nodes
     NAME                                        STATUS   ROLES    AGE    VERSION
     gke-vk-node-pool-vk-9e5f3d39-44rg           Ready    <none>   171m   v1.14.10-gke.36
-    virtual-kubelet                             Ready    agent    108s
+    kip                             Ready    agent    108s
 
 ## Run a Pod via Virtual Kubelet
 
-The node taint in virtual-kubelet is disabled in the manifest, so Kubernetes will try to run all pods via the virtual node.
+The node taint in kip is disabled in the manifest, so Kubernetes will try to run all pods via the virtual node.
 
-If you decide to enable the taint on the virtual node (via removing the `--disable-taint` command line flag), you will need to add a toleration and/or node selector for pods that are meant to run via virtual-kubelet:
+If you decide to enable the taint on the virtual node (via removing the `--disable-taint` command line flag), you will need to add a toleration and/or node selector for pods that are meant to run via kip:
 
     spec:
       nodeSelector:
-        type: virtual-kubelet
+        type: kip
```

Comment:
1. This label is also in node-cli, and we don't change it anywhere.
2. I think we can keep type as "virtual-kubelet". This is a common label used and shared by VK providers.